code/src/Components/InputTypeRadio.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export const InputTypeRadio = ({className, header, array, selected, setSelected}) => {
+  return(
+    <div className="favourite-color-container">
+      <h3 tabIndex="0">{header}</h3>        
+      <div className={className} aria-label="radiobutton group" tabIndex="0">            
+        {array.map((option) => (
+          <label key={option}>
+            <input 
+              type="radio"
+              value={option}
```

Comment:
Radio buttons should have boolean attribute `checked` instead of `value` - if you will test your website right now, you will see that we are able to check more than one radiobutton at the time. This is bug

Reply:
Fixed.